server/game/cards/08-GT/FranciscoRosales.js

```diff
@@ -0,0 +1,40 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+const StandardActions = require('../../PlayActions/StandardActions.js');
+
+class FranciscoRosales extends DudeCard {
+    setupCardAbilities(ability) {
+        this.job({
+            title: 'Francisco Rosales',
+            playType: 'noon',
+            cost: ability.costs.bootLeader(),
```

Comment:
This is not very intuitive, but in this case we should use `bootSelf()` cost because leader of the job is dude with the job ability. The `bootLeader()` cost is used only on other cards where player have to select a job leader.
```suggestion
            cost: ability.costs.bootSelf(),
```